index.js

```diff
@@ -203,3 +203,21 @@ menu.onclick = function close () {
   main.style.filter = 'blur(0)'
   menubtn.style.display = 'block'
 }
+/* -----------------Validation contact form ------------------- */
+const contactForm = document.getElementById('form')
+const error = document.getElementById('error')
+const emailRegex = /^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/
+
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault()
+
+  if (!emailRegex.test(contactForm.elements.email.value.trim())) {
+    error.innerText = 'Form not Sent !! \n Please enter your email in lower case.'
+    error.classList.add('contact-me-form-error-msg-display')
```

Comment:
[OPTIONAL] Well done 👏🏽 on displaying the error message, it really helps the user understand what's going on. I'd suggest improving it even further 💯 to use the `setTimeout` method to make the message disappear after an amount of time `5s` is correct because that's enough time for the user to read it. It would go like that:
```
    setTimeout(() => {
      error.classList.remove('contact-me-form-error-msg-display')
    }, 5000)
 ```